### ISSUE TITLE
Add option to change line colors

### ISSUE THREAD

**Issue by vajahath** [vajahath/lme#15]
# Opening discussion on configuring line color

We've provided option to customize colors for outputting logs with `lmeconfig.json` file. Currently it doesn't have option to change colors of line functions.

## What I'm proposing?
Using the log's color scheme for line functions may not be a good idea. Because, the combination of text-colors, background colors and modifiers (bold, italic etc.) may style logs pretty well, but not lines. For example, an `lmeconfig.json` file like below may produce good logs and bad lines:

```json
{
    "colors":{
        "error": ["red", "bgBlack", "underline"]
    }
}
```

So better option is to use separate configurations for lines and logs. So a configuration like below,

```json
{
    "colors":{
        "error": ["red", "bgBlack", "underline"]
    },
   "lines":{
       "error": ["red"]
   }
}
```

may produce better lines.

> **Let me know what others think.** I'm pushing the release date of lme 1.4 to another day to avoid any kind of potential breaking changes. And 1.4 will only be released when we confirm that it doesn't require any breaking changes to implement this.

**Comment by demacdonald** [comment]
hmm, I agree line log colors may not look great if they share the log colors...

what if:

```
{
    "colors":{
        "logs": {
               "error": ["red", "bgBlack", "underline"]
        },
        "lines":{
               "error": ["red"]
        }
    }
}
```
Or maybe more readable:

```
{
    "colors":{
           "error":  {
                 "logs": ["red", "bgBlack", "underline"],
                 "lines": ["red"]
            }
     }
}
```

mind you I typed this in github without a nice editor, so formatting might not be perfect, but you get the idea.

**Comment by vajahath** [comment]
```json
{
    "colors":{
        "logs": {
               "error": ["red", "bgBlack", "underline"]
        },
        "lines":{
               "error": ["red"]
        }
    }
}
```
Yeah. I liked this configuration.
- I think it is less complex
- with this config, we'll be able to add separate statuses/properties for lines and logs in future if required.

?

**Comment by demacdonald** [comment]
Yeah, works for me.

**Comment by vajahath** [comment]
ok 👍 I'll handle this.

**Comment by demacdonald** [comment]
Cool,  add me to the review :)

**Comment by vajahath** [comment]
This issue isn't yet ready to close. will fix this.

**Comment by vajahath** [comment]
Fixed by #23
